src/factions/AdeptaSororitas/abilities.json

```diff
@@ -0,0 +1,123 @@
+[
+    {
+        "Id": "SOF",
+        "ModelId": "BAS",
+        "NameEn": "Shield of Faith",
+        "NameFr": "Shield of Faith",
+        "NameDe": "Shield of Faith",
+        "DescriptionEn": "Models with this ability have a 6+ invulnerable save. In addition, models with this ability can attempt to deny one psychic power in each Psychic phase in the same manner as a Psyker. When making this attempt, roll one D6 instead of 2D6; the psychic power is resisted if the roll is greater than the result of the Psychic test that manifested the power.",
+        "DescriptionFr": "Models with this ability have a 6+ invulnerable save. In addition, models with this ability can attempt to deny one psychic power in each Psychic phase in the same manner as a Psyker. When making this attempt, roll one D6 instead of 2D6; the psychic power is resisted if the roll is greater than the result of the Psychic test that manifested the power.",
+        "DescriptionDe": "Models with this ability have a 6+ invulnerable save. In addition, models with this ability can attempt to deny one psychic power in each Psychic phase in the same manner as a Psyker. When making this attempt, roll one D6 instead of 2D6; the psychic power is resisted if the roll is greater than the result of the Psychic test that manifested the power.",
```

Comment:
If you made this jsonnet, you could use the translation function in Utils for a bunch of this.

Reply:
I need to checkout the jsonnet stuff (we should really get that into the README), but trying to get my head round that and the format etc was doing my head in so plain old JSON was easier. 

Will fix up when I get a better understanding of it.